src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetNativeVariantForObjectTests.cs

```diff
@@ -183,7 +163,6 @@ public static IEnumerable<object[]> GetNativeVariantForObject_NonRoundtrippingPr
         [Theory]
         [MemberData(nameof(GetNativeVariantForObject_NonRoundtrippingPrimitives_TestData))]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/26820")]
         public void GetNativeVariantForObject_ValidObject_Success(object primitive, VarEnum expectedVarType, IntPtr expectedValue, object expectedRoundtripValue)
         {
             var v = new Variant();
```

Comment:
`GetNativeVariant_NotInteropCompatible_TestData` method in this file is dead code.

Reply:
Actually so is `GetNativeVariantForObject_RoundtrippingPrimitives_TestData`.